pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -132,10 +140,10 @@ func IsUnrecoverableError(err error) bool {
 }
 
 type Queue struct {
-	workqueue.TypedRateLimitingInterface[*Command]
+	sync.RWMutex
```

Comment:
nit: we're not great about doing this everywhere, but generally we want to group the mutex with the resource it's protecting (in this case `nodeClaimToCommand`). I'd revert the ordering such that it's next to it.